### DISCUSSION TITLE
is there a way to set CHROME_PATH once on Ubuntu?

### DISCUSSION THREAD

**Discussion by amineHY** [marp-team/marp#368] [Q&A] [answered]
In order to generate a PDF file I have to set the CHROME_PATH every time I open a new session.

I a sure there is  way to set it once and forget it. I tried to add it to the ~.bashrc then source the file, but that didn't help

In my case I use Chrome based browser: Brave
CHROME_PATH=/user/bin/brave-browser

Thank you in advance

**Comment by yhatt** [comment, marked as answer]
https://help.ubuntu.com/community/EnvironmentVariables#Persistent_environment_variables
